### PR TITLE
Add default topologies for tests

### LINKED ISSUE #3662: TAS: Use ResourceFlavorWrapper and TopologyWrapper for testing
**What would you like to be cleaned**:

Use the wrappers to setup objects for TAS testing.

**Why is this needed**:

To make the code more consistent (as we use this strategy for other tests), and the shorter.

Example lengthy code: [here](https://github.com/kubernetes-sigs/kueue/blob/560e4340b746aa811cec06b7a072a1a3faedc80f/pkg/scheduler/scheduler_test.go#L4060-L4114).
See below the code for setting up ClusterQueue [here](https://github.com/kubernetes-sigs/kueue/blob/560e4340b746aa811cec06b7a072a1a3faedc80f/pkg/scheduler/scheduler_test.go#L4115-L4119)

### PR DESCRIPTION
**Pull request by mykysha**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Add default one, two and three level Topologies to be used in tests.

#### Which issue(s) this PR fixes:

Fixes #3662 

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/tas/statefulset_test.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ var _ = ginkgo.Describe("TopologyAwareScheduling for StatefulSet", func() {
 		gomega.Expect(k8sClient.Create(ctx, ns)).To(gomega.Succeed())
 
 		topology = testing.MakeTopology("datacenter").
-			Levels(topologyLevelBlock, topologyLevelRack, corev1.LabelHostname).
+			Levels(testing.DefaultBlockTopologyLevel, testing.DefaultRackTopologyLevel, corev1.LabelHostname).
 			Obj()
```
Comment (PBundyra) [inline test/e2e/tas/statefulset_test.go:50]: Let's use the new functions here

Comment (mykysha) [reply test/e2e/tas/statefulset_test.go]: Changed to `MakeDefaultThreeLevelTopology`

**Comment by mimowo** [comment]
/approve 
Leaving final lgtm to @PBundyra

**Comment by PBundyra** [comment]
/lgtm
Thanks @mykysha!!

**Pull request merged**